internal/runtime/catalog/catalog.go

```diff
@@ -40,24 +40,23 @@ type Catalog struct {
 	// gvhToType maps a GroupVersionHook to the corresponding RuntimeHook function.
 	gvhToType map[GroupVersionHook]reflect.Type
 
-	// gvhToType maps a RuntimeHook to the corresponding GroupVersionHook.
+	// typeToGVH maps a RuntimeHook function to the corresponding GroupVersionHook.
 	typeToGVH map[reflect.Type]GroupVersionHook
 
 	// gvhToHookDescriptor maps a GroupVersionHook to the corresponding hook descriptor.
 	gvhToHookDescriptor map[GroupVersionHook]hookDescriptor
 
-	// openAPIDefinitions maps a GroupVersion to a func
-	// which returns OpenAPI definitions for all request and response types with that
-	// GroupVersion.
+	// openAPIDefinitions is a list of OpenAPIDefinitionsGetter, which return OpenAPI definitions
+	// for all request and response types of a GroupVersion.
 	openAPIDefinitions []OpenAPIDefinitionsGetter
 
 	// catalogName is the name of this catalog. It is set based on the stack of the New caller.
 	// This is useful for error reporting to indicate the origin of the Catalog.
 	catalogName string
 }
 
-// Hook is a marker interface for a hook.
-// Hooks should be defined as a: func(*RequestType, *ResponseType).
+// Hook is a marker interface for a RuntimeHook function.
+// RuntimeHook functions should be defined as a: func(*RequestType, *ResponseType).
```

Comment:
```suggestion
// RuntimeHook functions should be defined as a: func <HoomName>(*RequestType, *ResponseType).
```

Reply:
Added a separate sentence. Not sure folks will understand the syntax otherwise :) (looks a bit like a generic)